_docs/concepts/auth.md

```diff
@@ -0,0 +1,154 @@
+<!----- Conversion time: 5.4 seconds.
+
+
+Using this Markdown file:
```

Comment:
I updated the header, and it's in the latest commit. I have some problem in running it locally. Will update later.